### ISSUE TITLE
I was wondering if it try to pull of the iOS 7 native email pull?

### ISSUE THREAD

**Issue by ghost** [yfujiki/YFJLeftSwipeDeleteTableView#1]
I was wondering if you can put more than one option just like the native email on iOS7?

**Comment by yfujiki** [comment]
Yeah, that is a good idea. And I can probably make the title flexible as well.

I will see if I can support additional multiple views.

**Comment by ghost** [comment]
sweet! I would play around with it also see what I can get!
